cmake/config.cmake

```diff
@@ -202,6 +202,6 @@ set(OMR_NOTIFY_POLICY_CONTROL OFF CACHE BOOL "TODO: Document")
 
 set(OMR_ENV_GCC OFF CACHE BOOL "TODO: Document")
 
-set(OMR_OPT_CUDA ${CUDA_FOUND} CACHE BOOL "Enable CUDA support in OMR")
+set(OMR_OPT_CUDA OFF CACHE BOOL "Enable CUDA support in OMR")
```

Comment:
Perhaps `OMR_CUDA_HOME` should be set (and documented) here?

Reply:
I think it makes more sense to keep `OMR_CUDA_HOME` in the `FindOmrCuda` module, but I could add a note to `OMR_OPT_CUDA` saying users should set `OMR_CUDA_HOME`. Would that be OK?

Reply:
I was thinking the `set(OMR_CUDA_HOME)` call that would appear here would also specify the type (directory or path?), but mentioning it in the description of `OMR_OPT_CUDA` works for me.

Reply:
OK, I've added the mention, and I've changed the type of `OMR_CUDA_HOME` to `PATH` (that means "path to a directory", it was just `STRING` before).